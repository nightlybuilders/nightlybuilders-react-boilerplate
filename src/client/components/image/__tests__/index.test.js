import React from 'react'
import { Image } from '../'

describe('Components/Image', () => {
  test('renders without throwing an error', () => {
    expect(
      <Image styles={{ width: 150 }} src={'http://via.placeholder.com/350x150'} alt="App" />,
    ).toMatchSnapshot()
  })
})
