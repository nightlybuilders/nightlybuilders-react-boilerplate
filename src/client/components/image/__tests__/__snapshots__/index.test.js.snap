// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Image renders without throwing an error 1`] = `
<Image
  alt="App"
  className=""
  onClick={[Function]}
  src="http://via.placeholder.com/350x150"
  styles={
    Object {
      "width": 150,
    }
  }
/>
`;
